### PR TITLE
Community maintenance 

### PR DESCRIPTION
**Pull request by jromero**
### Changes

- Make CONTRIBUTING more generic
- Add project policies to CONTRIBUTING
- Create DEVELOPMENT for more `pack` specific development help
- Add screen captured usage of `pack` to README
- Clean up README

> NOTE: This PR might be more readable by [browsing the branch in GH](https://github.com/buildpacks/pack/tree/feature/development-docs).

### Snapshot
image

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,102 +1,77 @@
...
+## Pull Request Process
 
-### Building
+1. [Fork][fork] the repository.
+2. Create a branch for the issue: `git checkout -b {{BRANCH_NAME}}`
+3. Make any changes deemed necessary.
+4. Commit your changes: `git commit -s`\
+    _Learn more about the [sign-off](#sign-off-process) process below._
+5. Push to GitHub: `git push origin {{BRANCH_NAME}}`
+6. [Create the pull request][create-pr].
```
Comment (natalieparellano) [inline CONTRIBUTING.md]: Should there be something here that links to the development section?

Comment (natalieparellano) [reply CONTRIBUTING.md]: Also we used to have some guidance around preferring PRs to have a single commit. Are we moving away from that?

Comment (jromero) [reply CONTRIBUTING.md]: > Also we used to have some guidance around preferring PRs to have a single commit. Are we moving away from that?

We are moving away from that. It ended up being to heavy handed. Some contributors prefer to "show their work".

Comment (jromero) [reply CONTRIBUTING.md]: > Should there be something here that links to the development section?

Good idea. I made a few changes. Want to look at it again?

**Comment by jromero** [comment]
_Depends on https://github.com/buildpacks/.github/pull/1_

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,102 +1,30 @@
...
-    git commit --amend --signoff
-    ```
-
-4. **Submit a Pull Request**
-
-    Submitting the pull request is done in [GitHub](https://github.com/buildpacks/pack/compare/) by selecting
-    your branch as the `compare` branch.
\ No newline at end of file
+[code-of-conduct]: https://github.com/buildpacks/.github/blob/master/CODE_OF_CONDUCT.md
+[contributing]: https://github.com/buildpacks/.github/blob/master/CONTRIBUTING.md
```
Comment (jromero) [inline CONTRIBUTING.md]: This link is broken until PR is merged: https://github.com/buildpacks/.github/pull/1

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,102 +1,82 @@
...
-| PACK_VERSION | Tell `pack` what version to consider itself                            | `dev`   |
+#### Preparing for a Pull Request
 
-[rgo]: https://github.com/kyoh86/richgo
+After making all the changes but before creating a [Pull Request][pull-request-process], you should run
+`make prepare-for-pr`. This command runs a set of other tasks that resolve or report any simple issues that would
+otherwise arise during the pull request review process.
 
-_NOTE: This project uses [go modules](https://github.com/golang/go/wiki/Modules) for dependency management._
+### User Acceptance on a Pull Request
```
Comment (natalieparellano) [inline CONTRIBUTING.md]: This is great!

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+### Testing
+
+To run unit and integration tests:
+```shell
+make unit
+```
+
+To run acceptance tests:
+```shell
+make acceptance
```
Comment (natalieparellano) [inline DEVELOPMENT.md]: Just a comment, I notice we have no mention of `make acceptance-all` (compat). This is probably for the best, until we finish our current work to download all the assets automatically. Maybe we can make documentation of that process a part of the current work.

File: README.md [hunk 5]
```diff
@@ -7,27 +7,34 @@
...
+- [**Buildpack Authors**][bp-author] to develop and package buildpacks for distribution.
+- [**Operators**][operator] to package buildpacks for distribution and maintain applications.
 
 ## Usage
 
-- [Installing][install-pack]
-- [Getting Started][getting-started]
+<img src="resources/pack-build.gif" width="600px" />
+
+## Getting Started
```
Comment (natalieparellano) [inline README.md]: Could we also link to buildpacks Slack?

File: DEVELOPMENT.md (hunk 4 above)
Comment (jromero) [reply DEVELOPMENT.md]: Definitely "for the best". I agree with your assessment.

File: README.md (hunk 5 above)
Comment (jromero) [reply README.md]: I like that idea! If we're ok, I'd like a new PR created for additional changes. I'm sure there's more we'd like to add but don't want to grow the scope of this indefinitely. 

Do you want to do the honors of that PR? I don't mind if you don't have the availability (or interest :D).

**Pull request merged**